multinode_integration_tests/src/masq_node.rs

```diff
@@ -476,7 +474,7 @@ mod tests {
 
     #[test]
     fn node_reference_can_display_itself() {
-        let chain_id = TEST_DEFAULT_MULTINODE_CHAIN_ID;
+        let chain_id = TEST_DEFAULT_MULTINODE_CHAIN;
```

Comment:
this still references `chain_id` - is this object supposed to be changed to `chain`??